testutils/proto_utils.go

```diff
@@ -0,0 +1,13 @@
+package testutils
+
+import "go.viam.com/utils/protoutils"
+
+// ToProtoMapIgnoreOmitEmpty is a helper to convert anything
+// to a map to compare against a structpb.
+func ToProtoMapIgnoreOmitEmpty(data any) map[string]any {
```

Comment:
I believe you meant to use an an `*anypb.Any` here, whereas `any` is a lesser known alias to `interface{}`. Since StructToStructPb takes in an `interface{}` and `AsMap()` returns a map[string]interface{}, would explicitly use interface{} here since it's much more conventional (and thus readable) in Go — or just explicitly go with anypb.Any.

Then just nit to update the comment above.

Reply:
I'll update it to be an interface. I meant to just take in any data without converting to an anypb, will update and push.